.github/workflows/build-ubuntu20.04-backwards-compatibility.yml

```diff
@@ -1,27 +1,12 @@
 name: build-ubuntu-20.04-backwards-compatibility
 on:
-  push:
-    branches:
-      - dev
-      - release-*
-      - refs/tags/*
-    paths-ignore:
-      - '.github/workflows/quarto-render.yml'
-      - '_quarto.yml'
-      - 'quarto-materials/*'
-      - '**/.md'
-      - 'doc/source/conf.py'
-      - 'tiledb/sm/c_api/tiledb_version.h'
-  pull_request:
-    branches:
-      - '*'  # must quote since "*" is a YAML reserved character; we want a string
-    paths-ignore:
-      - '.github/workflows/quarto-render.yml'
-      - '_quarto.yml'
-      - 'quarto-materials/*'
-      - '**/.md'
-      - 'doc/source/conf.py'
-      - 'tiledb/sm/c_api/tiledb_version.h'
+  workflow_call:
+    inputs:
+      job_name:
+        description: 'Name of the job for display purposes'
```

Comment:
How about a name here?

Reply:
This was leftover from some earlier parameterization that I decided against, will remove. The jobs have quite descriptive names in the UI right now:

https://github.com/TileDB-Inc/TileDB/runs/7675727198?check_suite_focus=true